www/src/components/MarkdownHeader/MarkdownHeader.module.scss

```diff
@@ -9,6 +9,7 @@
   justify-content: space-between;
   align-items: baseline;
   position: relative;
+  padding-right: 100px;
```

Comment:
This sort of thing shouldn't be necessary. That's why we use flexbox layout. The "edit" link is a column and the title is a column and flexbox will prevent them from overlapping. 😄 Using paddings this way makes the overall styles more brittle. Changing one thing breaks another.

The root of this problem is the absolutely positioned link. Couldn't we just do this instead?
```diff
diff --git a/www/src/components/MarkdownHeader/MarkdownHeader.module.scss b/www/src/components/MarkdownHeader/MarkdownHeader.module.scss
index 3214da557..07a3a14aa 100644
--- a/www/src/components/MarkdownHeader/MarkdownHeader.module.scss
+++ b/www/src/components/MarkdownHeader/MarkdownHeader.module.scss
@@ -9,7 +9,6 @@
   justify-content: space-between;
   align-items: baseline;
   position: relative;
-  padding-right: 100px;
 }
 
 .Title {
@@ -28,9 +27,6 @@
   transition: all 0.2s ease;
   transition-propery: color, border-color;
   white-space: nowrap;
-  position: absolute;
-  bottom: 5px;
-  right: 0;
 
   &::after {
     top: -5px;
```